hugo/layouts/_default/baseof.html

```diff
@@ -4,20 +4,34 @@
     <body>
         {{- partial "navbar.html" . -}}
         {{- partial "banner.html" . -}}
-        {{- partial "breadcrumbs.html" . -}}
+
+        {{ $isGenomePage := eq .RelPermalink "/genome-browser/" }}
+        {{ if not $isGenomePage }}
+            {{- partial "breadcrumbs.html" . -}}
+        {{ end }}
+
         {{- block "header" . }}{{- end }}
         <main>
-          <div class="container" id="content">
-                  {{ if .Params.toc }}
-                          <div class="row">
-                              <div class="col-md-3 order-md-first toc small">{{- partial "toc.html" . -}}</div>
-                              <div class="col-md-9 order-md-last content" id="content">{{- block "main" . }}{{- end }}</div>
-                          </div>
-                  {{ else }}
-                          {{- block "main" . }}{{- end }}
-                  {{ end }}
-          </div>
+
+            {{ if $isGenomePage }}
```

Comment:
To avoid the multiplication of conditional guards, it would be possible to define a dedicated layout used by the browser page: `layouts/browser/baseof.html`  But that refactoring could be attempted in another PR.

Reply:
Yes, that sounds good. I suppose we can do this once we have feedback about exactly what should be included in the browser page (header/footer wise).